bootstrap.d/sys-apps.y4.yml

```diff
@@ -1418,3 +1418,117 @@ packages:
           DESTDIR: '@THIS_COLLECT_DIR@'
         quiet: true
       - args: ['touch', '@THIS_COLLECT_DIR@/usr/etc/udev/rules.d/.keep']
+
+  - name: systemd-utils
+    labels: [aarch64, riscv64]
```

Comment:
If we enable this for all architectures, one should also enable libcap for them.